### PR TITLE
Minor improvements to command line args

### PR DESCRIPTION
**Pull request by nhojb**
* Adds a --help flag and slightly clearer usage message.
* Permits processing more than 1 storyboard file (or parent directory) via cmd line args.

### CODE REVIEW

File: Sources/natalie/main.swift [hunk 1]
```diff
@@ -8,18 +8,28 @@
...
-    filePaths = s
+
+for arg in CommandLine.arguments {
+    if arg == "--help" {
+        printUsage()
+        exit(0)
+    } else if arg.hasSuffix(storyboardSuffix) {
+        filePaths.append(arg)
+    } else if let s = findStoryboards(rootPath: arg, suffix: storyboardSuffix) {
+        filePaths.append(contentsOf: s)
```
Comment (krzyzanowskim) [inline Sources/natalie/main.swift:31]: append? currently, it replaces path meaning if the absolute path is provided it's not broken.

**Review by krzyzanowskim — CHANGES_REQUESTED** [review]
argument path is no longer absolute? It can support both but should't change the current behaviour I'd say

**Comment by nhojb** [comment]
Logic is unchanged from previously AFAICT. But rather than only processing arguments[1] (first path) the loop will process all paths (relative or absolute). Either way you end up with an array of paths to process - just like you do if you call "findStoryboards" on a directory arg.

**Review by krzyzanowskim — APPROVED** [review]

**Comment by krzyzanowskim** [comment]
thanks, looks good. Good job!

**Pull request merged**